licenses/print.py

```diff
@@ -0,0 +1,143 @@
+from .models import LicenseRequest
+from PyPDF2 import PdfReader
+from PyPDF2 import PdfWriter
+from datetime import date
+from django.conf import settings
+from django.http import FileResponse
+from django.utils.translation import gettext as _
+from reportlab.pdfgen import canvas
+import io
+
+
+def generate_license_file(user, lr: LicenseRequest) -> FileResponse:
+    """
+    Generate a License as pdf file.
+
+    As template the '2017_Antrag_Einzelgenehmigung_ausfuellbar.pdf' from
+    https://www.okmq.de/images/Formulare/2017_Antrag_Einzelgenehmigung_ausfuellbar.pdf
+    is used.
+    The function assumes that the LicenseRequest has a user with profile.
+    """
+    def f(p): return str(p) if p else '  -------  '
```

Comment:
This function does not to be nested and I would be happy not to have it as a one-liner.